src/pgroonga-crash-safer.c

```diff
@@ -845,6 +851,21 @@ _PG_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomIntVariable("pgroonga_crash_safer.max_recovery_threads",
+							"Maximum number of threads for recovery of broken Groonga indexes.",
+							"The default is 0, which means disabled. "
+							"Use all CPUs in the environment at -1. "
+							"Use CPU for that number if max_recovery_threads >= 1 is set.",
```

Comment:
```suggestion
							"Use CPU for that number if 1 or later is set.",
```

or `pgroogna_crash_safer.max_recovery_threads`.